src/index.html

```diff
@@ -20,8 +20,380 @@
       href="styles/index.scss"
     />
```

Comment:
The link to the stylesheet is incorrect. You are linking to a `.scss` file, but the browser does not understand SCSS syntax. You should compile your SCSS to CSS and then link to the compiled CSS file.